.dev/.env.dist

```diff
@@ -0,0 +1,27 @@
+RAILS_ENV=development
```

Comment:
Can we move the files to `.dev/docker/` perhaps? That may be easier to identify.